packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -5391,77 +5391,114 @@ void main() {
     expect(focusNode.hasFocus, false);
   });
 
-  testWidgets('TextEditingController.clear() behavior test', (WidgetTester tester) async {
-    // Regression test for https://github.com/flutter/flutter/issues/66316
-    final List<MethodCall> log = <MethodCall>[];
-    SystemChannels.textInput.setMockMethodCallHandler((MethodCall methodCall) async {
-      log.add(methodCall);
+  group('TextEditingController', () {
```

Comment:
This diff is confusing, but what I did was add the test on the next line and group it with the following existing test.